src/main/java/com/ldtteam/structurize/client/gui/WindowExtendedBuildTool.java

```diff
@@ -466,6 +466,20 @@ public void settingsClicked()
         folderList.hide();
         blueprintList.disable();
         blueprintList.hide();
+
+        final View cats = findPaneOfTypeByID("categories", View.class);
```

Comment:
What does this do?

Reply:
Hides categories when settings are displayed to avoid visual clutter